flask/app/routes.py

```diff
@@ -232,3 +234,23 @@ def update_entity(model_name:str, id:int):
 
     return jsonify(table)
 
+@app.route('/api/participant', methods=['GET'], endpoint='partipants_list')
+@login_required
+def participants_list():
+    db_participants = models.Participant.query.options(
+        joinedload(models.Participant.family),
+        joinedload(models.Participant.tissue_samples).
+        joinedload(models.TissueSample.datasets)
+    ).all()
+    return jsonify([
+        {
+            **asdict(participant),
+            'family_codename': participant.family.family_codename,
+            'tissue_samples': [
+                {
```

Comment:
Might be just preference but collapse the outside brackets of the dictionaries and lists for readability since there's a lot going on here?

Reply:
What do you think of this approach Delvin?

Reply:
I like it, it's a clean solution that doesn't require external packages or references to a million fields